samples/matter/thermostat/README.rst

```diff
@@ -272,17 +275,14 @@ After building this sample and the :ref:`Matter weather station <matter_weather_
 
    .. code-block:: console
 
-      $ ./chip-tool accesscontrol write acl '[{"fabricIndex": 1, "privilege": 5, "authMode": 2, "subjects": [112233], "targets": null}, \
-      {"fabricIndex": 1, "privilege": 1, "authMode": 2, "subjects": [<thermostat_node_ID>], "targets": \
-      [{"cluster": 1026, "endpoint": <thermostat_sensor_endpoint_ID>, "deviceType": null}]}]' <Sensor Device NodeId> 0
+      $ ./chip-tool accesscontrol write acl '[{"fabricIndex": 1, "privilege": 5, "authMode": 2, "subjects": [112233], "targets": null}, {"fabricIndex": 1, "privilege": 1, "authMode": 2, "subjects": [<thermostat_node_ID>], "targets": [{"cluster": 1026, "endpoint": <temperature_sensor_endpoint_ID>, "deviceType": null}]}]' <Sensor Device NodeId> 0
```

Comment:
Could we change `<Sensor Device NodeId>` to `<temperature_sensor_node_ID>` or something similar to follow the convention? Also could we mention this parameter in the bullet point description?

Reply:
good point

Reply:
Btw. I meant also adding `<temperature_sensor_node_ID>` to the following text: 

> Use the following command, with *<thermostat_node_ID>* and *<temperature_sensor_endpoint_ID>* values from the previous step about commissioning: